prisma/schema.prisma

```diff
@@ -12,3 +12,15 @@ datasource db {
   provider = "postgresql"
   url      = env("DATABASE_URL")
 }
+
+model Contrato {
+  id          Int      @id @default(autoincrement())
+  numero      String   @unique @db.VarChar(10)
+  nomeGestor  String   @db.VarChar(255)
+  descricao   String   @db.VarChar(255)
+  dataInicio  DateTime @default(now())
+  dataTermino DateTime @default(now())
+  ativo       Boolean  @default(true)
```

Comment:
O default do campo ativo não deveria ser false ao invés de true porque ele fica inativo automaticamente ?

Reply:
No meu entendimento, quando se cria o contrato por padrão se entende que ele ficará ativo, porque se não ao criar o contrato mesmo que com uma data futura, o usuario tera que ativar o contrato manualmente toda vez.

O contrato so ficara automaticamente inativo quando a data de termino for a data do dia, mas essa logica sera implementada ainda nas features seguintes.

Reply:
Beleza.